src/adapters/default/config.adapter.ts

```diff
@@ -0,0 +1,183 @@
+import { HttpService } from "@nestjs/axios";
+import { Injectable, HttpException } from "@nestjs/common";
+const resolvePath = require("object-resolve-path");
+import { AxiosResponse } from "axios";
+import { ConfigDto } from "src/configs/dto/config.dto";
+import { first, map, Observable } from "rxjs";
+import { SuccessResponse } from "src/success-response";
+import { catchError } from "rxjs/operators";
+import { ErrorResponse } from "src/error-response";
+import { ConfigSearchDto } from "src/configs/dto/config-search.dto";
+import jwt_decode from "jwt-decode";
+import { TeacherDto } from "../../teacher/dto/teacher.dto";
+@Injectable()
+export class ConfigService {
+  constructor(private httpService: HttpService) {}
+  url = `${process.env.BASEAPIURL}/Config`;
+
+  public async createConfig(request: any, configDto: ConfigDto)
+  {
+    let axios = require("axios");
+    let data = {
+      filters: {
+        module: {
+          eq: `${configDto.module}`,
+        },
+        key: {
+          eq: `${configDto.key}`,
+        },
+        contextId: {
+          eq: `${configDto.contextId}`,
+        },
+      },
+    };
+
+    let config = {
+      method: "post",
+      url: `${this.url}/search`,
+
+      data: data,
+    };
+
+    const response = await axios(config);
+    let resData = response?.data;
+    let result = resData.map((item: any) => new ConfigDto(item));
+
+    let configId = result.map(function (ConfigDto) {
+      return ConfigDto.configId
+    })
+
+    if (resData.length > 0)
+    {
+      var udateData = configDto;
+      var updateConfig = {
+      method: "put",
+      url: `${this.url}/${configId}`,
+      headers: {
+        Authorization: request.headers.authorization,
+      },
+      data: udateData,
+      };
+      const response = await axios(updateConfig);
+      console.log("123");
```

Comment:
Remove these logging statements please. Instead you can print a proper log message that can be used for debugging later.